app/_hub/nonamesecurity/nonamesecurity-kongprevention/overview/_index.md

```diff
@@ -0,0 +1,125 @@
+---
+nav_title: Overview
+---
+
+The Noname Security prevention integration for Kong Open Source or Kong Enteprise instructs the Kong API Gateway to block API requests that are associated with anomalous IP addresses, headers, cookies, query strings, or JWT tokens. Kong Konnect data-plane integration for this noname machine learning powered prevention integration is currently in BETA. 
```

Comment:
```suggestion
The Noname Security prevention integration for {{site.base_gateway}} instructs the Gateway to block API requests that are associated with anomalous IP addresses, headers, cookies, query strings, or JWT tokens.

{:.note}
 > {{site.konnect_product_name}} data plane integration for this integration is currently in BETA. 
```
`{{site.base_gateway}}` will resolve to Kong Gateway (the general name for either OSS or Enterprise), and `{{site.konnect_product_name}}` will resolve to Kong Konnect.

You don't need to say the full plugin name every time.